src/odemis/gui/plugin/__init__.py

```diff
@@ -409,7 +409,7 @@ def addButton(self, label, callback=None, face_colour='def'):
                               style=wx.ALIGN_CENTER, face_colour=face_colour)
         self.buttons.append(btn)
         sizer = self.pnl_buttons.GetSizer()
-        sizer.Add(btn, proportion=1, flag=wx.ALL | wx.ALIGN_RIGHT, border=10)
+        sizer.Add(btn, proportion=1, flag=wx.ALL, border=10)
```

Comment:
why is align_right removed here?

Reply:
Good question. I've checked. It's another rule, that I forgot to indicate in the commit log: as the sizer is horizontal, it's not possible to specify horizontal alignment. I've updated the commit log.